### ISSUE TITLE
React compatibility: Use new createRoot API

### ISSUE THREAD

**Issue by coder-karen** [Automattic/jetpack#38480]
## Issue

In React 18, importing `createRoot` from `react-dom` was deprecated, so ideally we'll fix up instances of this.

Related warnings:
```
Warning: You are importing createRoot from "react-dom" which is not supported. You should instead import it from "react-dom/client". visible on the Jetpack settings page.
```

Parent React 19 compatibility issue: https://github.com/Automattic/jetpack/issues/37951

**Comment by anomiex** [comment]
Looks like just one instance was added since this was previously fixed in #28710.
https://github.com/Automattic/jetpack/blob/d17273c495482eda4af2dd310cb8072c63c6c94d/projects/js-packages/components/components/boost-score-graph/tooltips-plugin.ts#L26

**Comment by coder-karen** [comment]
Thanks! Looks like there are two separate issues related to `createRoot`, so I'll address them both.
Edit - no - just the one, thanks for clarifying in the PR!
